src/Lecture1.hs

```diff
@@ -63,7 +65,8 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = if n < 0 then (-1 * n) `mod` 10 else n `mod` 10
```

Comment:
That's correct! 💯 
And you also use the infix notation using backticks for `mod` 😍 

You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```